src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -46,13 +48,28 @@
 ERROR_MESSAGE_DRIVER_NOT_CONNECTED = """
 [Driver] Error: Not connected.
 
-Call `connect()` on the `GrpcDriverHelper` instance before calling any of the other
-`GrpcDriverHelper` methods.
+Call `connect()` on the `GrpcDriverStub` instance before calling any of the other
+`GrpcDriverStub` methods.
 """
 
 
-class GrpcDriverHelper:
-    """`GrpcDriverHelper` provides access to the gRPC Driver API/service."""
+class GrpcDriverStub:
+    """`GrpcDriverStub` provides access to the gRPC Driver API/service.
+
+    Parameters
+    ----------
+    driver_service_address : Optional[str]
+        The IPv4 or IPv6 address of the Driver API server.
+        Defaults to `"[::]:9091"`.
+    certificates : bytes (default: None)
```

Comment:
Both name and type are not in line with the name/type in `__init__`. The description is not in line with either of them, it suggests that the type would have to be `Optional[Tuple[bytes, bytes, bytes]]`.

Reply:
Oh, yes. Thanks for pointing it out. I just moved the docstring from the `GrpcDriver`. Didn't notice that it was wrong originally.